cns/restserver/api_test.go

```diff
@@ -1292,6 +1292,37 @@ func TestCreateHostNCApipaEndpoint(t *testing.T) {
 	fmt.Printf("createHostNCApipaEndpoint Responded with %+v\n", createHostNCApipaEndpointResponse)
 }
 
+func TestNCIDCaseSensitive(t *testing.T) {
+	setEnv(t)
+	err := setOrchestratorType(t, cns.Kubernetes)
+	if err != nil {
+		t.Fatalf("TestNCIDCaseSensitive failed with error:%+v", err)
+	}
+
+	// test both upper-case NCIDs and cns-managed mode NCID
+	upperCaseNCID1 := "Swift_89063DBF-AA31-4BFC-9663-3842A361F188"
```

Comment:
why do you need to create these vars?
Does this work?
`ncids := []string{Swift_89063DBF-AA31-4BFC-9663-3842A361F188, Swft_XXXX, YYYY, Swift_lowercase}`